<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogSolverSLE</class>
 <widget class="QDialog" name="DialogSolverSLE">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1375</width>
    <height>640</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QComboBox" name="comboBoxRows">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>20</y>
     <width>161</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>60</y>
     <width>861</width>
     <height>461</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="getDataButton">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>20</y>
     <width>201</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>get data</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="solutionTableWidget">
   <property name="geometry">
    <rect>
     <x>1020</x>
     <y>60</y>
     <width>256</width>
     <height>461</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="solveButton">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>20</y>
     <width>181</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>solve</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>10</y>
     <width>231</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Choose number of unknowns</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>1110</x>
     <y>19</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Solutions</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
